main.ts

```diff
@@ -109,9 +133,85 @@ async function run(): Promise<void> {
     })) {
       ln(`/dev/${linkPath}`, `/proc/self/${target}`)
     }
+
+    if (msystem === 'CLANGARM64') {
+      // ARM64 dependencies aren't included yet in the Git for Windows SDK. Ask Pacman to install them.
+      core.startGroup(`Installing CLANGARM64 dependencies`)
+      await installArm64Dependencies(outputDirectory)
+      core.endGroup()
+    }
```

Comment:
Should I only do this part when `flavor === 'full'`?

Reply:
What are they used for? If they are required to build the Pacman package, `flavor === 'makepkg'` might need them, too, and `flavor === 'build-installers'` probably needs them, too.

Reply:
Yes, those dependencies are always needed for building Git on Windows ARM64, both directly from `git-for-windows/git` and as the Pacman package `mingw-w64-git`. It basically ensures that things like `zlib1.dll` and the Clang compiler are available in `/clangarm64`. That means we always need it, right?

Reply:
> That means we always need it, right?

Right, even for the `minimal` case, which is a bit sad because the `installArm64Dependencies()` call is prone to take a while. But it is what it is, right?

Reply:
@dennisameling oh my, I completely bungled this advice. We _cannot_ install the ARM64 dependencies in the `minimal` case because there is no `pacman` in the `minimal` subset...

Reply:
Thanks for the follow-up! Let's zoom out a bit. Is this a good description of the various flavors for ARM64 specifically?

- `minimal` - Based on your comment, I don't think this flavor makes a lot of sense at all for ARM64 at this point. I've only been using `full` and `makepkg-git`. Not ideal, but good enough to get a first working version I think. I saw that you already throw an error if users attempt to use `aarch64` in combination with this flavor, which makes perfect sense to me.
- `makepkg-git` - contains `mingw-w64-clang-aarch64-toolchain` and `mingw-w64-clang-aarch64-asciidoc` for building Git, in addition to the runtime `aarch64` dependencies. I like that this flavor is cached by default - it speeds things up a bit. I'll test this flavor more thoroughly once we get https://github.com/git-for-windows/MINGW-packages/pull/56 over the finish line.
- `build-installers` - will only install the runtime `aarch64` dependencies, needed for building artifacts like `portable`. Ensures that binaries like `clang.exe` won't leak into the final packages, without having to explicitly exclude them in [`make-file-list.sh`](https://github.com/git-for-windows/build-extra/blob/main/make-file-list.sh)
- `full` - slight overkill for most CI scenarios I guess, but: it works as expected.

Reply:
That's a good description, I think.

Just pointing out one thing: `build-installers`, in the non-ARM64 case, is supposed to fully include `makepkg-git`. It is the role of [`make-file-list.sh`](https://github.com/git-for-windows/build-extra/blob/main/make-file-list.sh) to ensure that no build dependencies leak out into the installers.